libpod/runtime.go

```diff
@@ -500,6 +501,15 @@ func makeRuntime(ctx context.Context, runtime *Runtime) (retErr error) {
 		// no containers running.  Create immediately a namespace, as
 		// we will need to access the storage.
 		if needsUserns {
+			// warn users if mode is rootless and cgroup manager is systemd
+			// and no valid systemd session is present
+			// warn only whenever new namespace is created
+			if runtime.config.Engine.CgroupManager == config.SystemdCgroupsManager {
+				unified, _ := cgroups.IsCgroup2UnifiedMode()
+				if unified && rootless.IsRootless() && !utils.CheckifSystemdSessionisValid(rootless.GetRootlessUID()) {
+					logrus.Infof("Invalid systemd user session for current user")
```

Comment:
This should be higher the Infof, most likely Warning.
Info does not show by default, so likely no one will see this.

Reply:
@rhatdan changed to warning instead :)